libs/community/langchain_community/chains/graph_qa/cypher.py

```diff
@@ -65,7 +65,17 @@ def extract_cypher(text: str) -> str:
     # Find all matches in the input text
     matches = re.findall(pattern, text, re.DOTALL)
 
-    return matches[0] if matches else text
+    # Extracted Cypher query
+    cypher_query = matches[0] if matches else text
+
+    # Replace node labels with backticks if they contain spaces
+    cypher_query = re.sub(
```

Comment:
we can simplify it be always quoting all labels, rel-types and property keys and removing backticks from them.